examples/ngrok-tunnel/README.md

```diff
@@ -1,10 +1,10 @@
 # Example: ngrok tunnel example
 
-Ngrok is a handy solution to expose your local environment to the Web. You can get information about this third-party solution on their official web page: https://ngrok.com.
+[Ngrok](https://ngrok.com) is a handy http tunnel you can use to expose your local environment to the Web and inspect incoming requests.
 
 ## Test the example
 
-1. First you will have to Sign up (it's free). Once it's done, on the left menu clic on "Getting Started > Your Authtoken".
```

Comment:
Ngrok is not free for everyone / every use case.

Reply:
It is free for most common use cases. I'm trying to convince the enduser to try it

Reply:
I added this sentence: `For this simple use case the free plan is sufficient.`